spec/frodo/entity_spec.rb

```diff
@@ -59,8 +59,8 @@
     let(:properties) { {
       "firstname"                => "Christoph",
       "lastname"                 => "Wagner",
-      "ownerid"                  => "odata.bind@/systemusers(95B9F1A8-3D5A-E911-A956-000D3A3B9CD8)",
-      "parentcustomerid_account" => "odata.bind@/accounts(60fb3f1c-b766-e911-a955-000d3a3b9316)",
+      "ownerid@odata.bind"                  => "/systemusers(95B9F1A8-3D5A-E911-A956-000D3A3B9CD8)",
```

Comment:
This test was just completely wrong before-- that's not how you do annotations.

Reply:
makes sense